test/blackbox-tests/test-cases/melange/merlin.t

```diff
@@ -3,7 +3,10 @@
   $ ocamlc_where="$(ocamlc -where)"
   $ export BUILD_PATH_PREFIX_MAP="/OCAMLC_WHERE=$ocamlc_where:$BUILD_PATH_PREFIX_MAP"
   $ melc_where="$(melc -where)"
-  $ export BUILD_PATH_PREFIX_MAP="/MELC_WHERE=$melc_where:$BUILD_PATH_PREFIX_MAP"
+  $ IFS=: read -r melc_stdlib melc_stdlib2 melc_stdlib3 <<< "$melc_where"
+  $ export BUILD_PATH_PREFIX_MAP="/MELC_WHERE1=$melc_stdlib:$BUILD_PATH_PREFIX_MAP"
+  $ export BUILD_PATH_PREFIX_MAP="/MELC_WHERE2=$melc_stdlib2:$BUILD_PATH_PREFIX_MAP"
+  $ export BUILD_PATH_PREFIX_MAP="/MELC_WHERE3=$melc_stdlib3:$BUILD_PATH_PREFIX_MAP"
```

Comment:
not happy about this at all. Here's the problem:

- `BUILD_PATH_PREFIX` is `:` separated
- the result of `melange --where` is `:` separated

Happy to hear alternatives. In the meantime this makes the tests pass.

Reply:
Looks fine to me. I don't see  the problem apart from hard coding it to use 3 paths. I would also substitute them all to `MELC_STDLIB`.